apps/notifications/src/panel/templates/summary-in-single.jsx

```diff
@@ -61,11 +68,55 @@ class UserHeader extends Component {
 	}
 }
 
+class WritingPromptHeader extends Component {
+	render() {
+		const icon = this.props.site.media[ 0 ];
+		const img_tag = (
+			<img src={ icon.url } height={ icon.height } width={ icon.width } alt="prompt-icon" />
```

Comment:
`alt="prompt-icon"` should be something human readable and translated

Reply:
Removing alt values as this is what is done elsewhere - its only added to avoid lint issue and not sure any value populating these images with alt property.